.github/workflows/pr.yml

```diff
@@ -93,7 +92,7 @@ jobs:
           retention-days: 1
 
   build-jvm-linux-native-lib:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
What version of `libc` it has? the goal with old `centos:7` is to build it on the smallest libc Core supports. So maybe we can still use/install the old libc via ubuntu